libraries/botframework-connector/src/teams/models/mappers.ts

```diff
@@ -270,12 +270,18 @@ export const TeamsParticipantChannelAccount: msRest.CompositeMapper = {
         className: 'TeamsParticipantChannelAccount',
         modelProperties: {
             ...TeamsChannelData.type.modelProperties,
-            memberRole: {
+            meetingRole: {
                 serializedName: 'memberRole',
```

Comment:
should this be 'meetingRole'?

Reply:
I think I must have fixed this while you were reviewing 😄